src/lib/chat/chat-message.ts

```diff
@@ -70,100 +72,114 @@ function getMediaPreview(message: Message) {
   }
 }
 
-export function adminMessageText(message: Message, state: RootState) {
-  const user = currentUserSelector()(state);
-
+export function adminMessageText(message: Message, currentUserId: string, getUser: (id: string) => User | undefined) {
```

Comment:
We were passing root state through a bunch of utils when really we just needed a getUser method